Gemfile.lock

```diff
@@ -305,6 +310,7 @@ GEM
       mime-types (>= 1.16, < 4.0)
       netrc (~> 0.8)
     rexml (3.2.5)
+    rinku (2.0.6)
```

Comment:
Does auto_html depend on rinku or did this sneak in from your first pass?

Reply:
heh yeah interestingly it's also a dependency from `auto_html` :smile: